ios/Classes/SwiftTorusDirectPlugin.swift

```diff
@@ -9,134 +9,239 @@ struct TorusDirectArgs {
-                code: "MissingArgumentException",
-                message: "Missing triggerLogin arguments",
-                details: nil))
-            return
-        }
-        let subVerifierDetails = SubVerifierDetails(
-            loginType: .web,
-            loginProvider: .jwt,
-            clientId: "<empty>",
-            verifierName: verifier,
-            redirectURL: initArgs.redirectUri,
-            browserRedirectURL: "https://scripts.toruswallet.io/redirect.html",
-            extraQueryParams: [:],
-            jwtParams: [:]
-        )
-        let torusDirectSdk = TorusSwiftDirectSDK(
-            aggregateVerifierType: .singleLogin,
-            aggregateVerifierName: verifier,
-            subVerifierDetails: [subVerifierDetails]
-        )
-        torusDirectSdk.getTorusKey(verifier: verifier, verifierId: verifierId, idToken: idToken, userData: verifierParams).done { data in
-            result(data)
-        }.catch { err in
-            result(FlutterError())
+        switch call.method {
+        case "init":
+            guard
+                let network = args["network"] as? String,
+                let browserRedirectUri = args["browserRedirectUri"] as? String,
+                let redirectUri = args["redirectUri"] as? String
+            else {
+                result(FlutterError(
+                        code: "MISSING_ARGUMENTS",
+                        message: "Missing init arguments",
+                        details: nil))
+                return
+            }
+            self.torusDirectArgs = TorusDirectArgs(
+                network: network, browserRedirectUri: browserRedirectUri, redirectUri: redirectUri)
+            print("TorusDirectPlugin#init: " +
+                    "network=\(network), " +
+                    "browserRedirectUri=\(redirectUri), redirectUri=\(redirectUri)")
+            result(nil)
+        case "triggerLogin":
+            guard let initArgs = self.torusDirectArgs
+            else {
+                result(FlutterError(
+                        code: "NotInitializedException",
+                        message: "TorusDirect.init has to be called first",
+                        details: nil))
+                return
+            }
+            guard
+                let typeOfLogin = args["typeOfLogin"] as? String,
+                let verifier = args["verifier"] as? String,
+                let clientId = args["clientId"] as? String
+            else {
+                result(FlutterError(
+                        code: "MissingArgumentException",
+                        message: "Missing triggerLogin arguments",
+                        details: nil))
+                return
+            }
+            guard let loginProvider = LoginProviders(rawValue: typeOfLogin) else {
+                result(FlutterError(
+                        code: "InvalidTypeOfLogin",
+                        message: "Invalid type of login",
+                        details: nil))
```

Comment:
All error codes are ending with `..Exception`, I think this should share the same pattern too.